cmd/mcu_com.c

```diff
@@ -9,6 +9,16 @@ __weak int board_mcu_set_cpu_ready(void)
 	return 0;
 }
 
+__weak int get_leica_board_id(void)
+{
+	return 0;
+}
+
+__weak int get_production_flag(void)
+{
+	return 0;
+}
```

Comment:
Functions return _OK_ even if they do not do anything? Perhaps `-1`?

Reply:
I agree.
I take the chance to discuss about what we can do if we don't receive the board id.
Right now we have a default 'fit_config' environment variable that is not overwritten, if we don't receive the board id, and we boot with a pt1 configuration (dtb).
What can we do in the final implementation? Shall we decide not to boot at all if there is no MCU response?